File: Exemplos M10/App_IntentDigitalHUB_Ionic/Elgin_IntentDigitalHUB_Ionic-font/src/intentDigitalHubService/SAT/Commands/ConsultarStatusOperacional.ts
import { SatCommand } from "../SatCommand";

export class ConsultarStatusOperacional extends SatCommand {
  readonly numSessao: number;
  readonly codAtivacao: string;

  constructor(numSessao: number, codAtivacao: string) {
    super("ConsultarStatusOperacional");
    this.numSessao = numSessao;
    this.codAtivacao = codAtivacao;
  }

  functionParameters(): object {
    return { numSessao: this.numSessao, codAtivacao: this.codAtivacao };
  }
}

File: Exemplos M10/App_IntentDigitalHUB_XamarinAndroid/Elgin_IntentDigitalHUB_XamarinAndroid-font/IntentServices/Sat/ConsultarStatusOperacional.cs
namespace Xamarin_Android_Intent_Digital_Hub.IntentServices.Sat
{
    class ConsultarStatusOperacional : SatCommand
    {
        readonly int numSessao;
        readonly string codAtivacao;

        public ConsultarStatusOperacional(int numSessao, string codAtivacao) : base("ConsultarStatusOperacional")
        {
            this.numSessao = numSessao;
            this.codAtivacao = codAtivacao;
        }

        protected override string FunctionParameters()
        {
            return "\"numSessao\"" + ":" + numSessao + "," +
                    "\"codAtivacao\"" + ":" + "\"" + codAtivacao + "\"";
        }
    }
}